bedrock-devnet/devnet/__init__.py

```diff
@@ -159,24 +161,16 @@ def devnet_l1_genesis(paths):
     log.info('Generating L1 genesis state')
     init_devnet_l1_deploy_config(paths)
 
-    geth = subprocess.Popen([
-        'anvil', '-a', '10', '--port', '8545', '--chain-id', '1337', '--disable-block-gas-limit',
-        '--gas-price', '0', '--base-fee', '1', '--block-time', '1'
-    ])
+    fqn = 'scripts/Deploy.s.sol:Deploy'
+    run_command([
+        'forge', 'script', '--chain-id', '1337', fqn, "--sig", "runWithStateDump()"
```

Comment:
I believe the chain id for op-e2e is 901 - perhaps we can use that as the chainid here to be more realistic

Reply:
Would like a double check on that value as being correct

Reply:
Chain id for L1 is `900`, I've changed it to that from `1337`

Reply:
awesome, thank you!